tools/ci_build/github/azure-pipelines/templates/react-native-ci.yml

```diff
@@ -59,30 +65,44 @@ jobs:
   - script:
       brew install coreutils ninja npm yarn
     displayName: Install coreutils, ninja, npm, and yarn
+  
+  - script:
+      yarn global add detox-cli
+    displayName: Install detox cli tools
+  
+  - script:
+      brew tap wix/brew
+    displayName: brew tap wix/brew
+  
+  - script:
+      brew install applesimutils
+    displayName: Install applesimutils tools required by detox ios
 
   - script: |
-      python3 -m pip install -q flatbuffers
+      python -m pip install -q flatbuffers
     displayName: Install python modules
 
   - script: |
       set -e -x
-      $(Build.SourcesDirectory)/tools/ci_build/github/linux/docker/inference/x64/python/cpu/scripts/install_protobuf.sh -d $(Build.SourcesDirectory)/cmake/deps.txt -p $(Build.BinariesDirectory)/installed      
-      python3 $(Build.SourcesDirectory)/tools/ci_build/github/apple/build_ios_framework.py \
-        --config ${{parameters.BuildConfig}} \
-        --build_dir $(Build.BinariesDirectory)/ios_framework --path_to_protoc_exe $(Build.BinariesDirectory)/installed/bin/protoc \
-        $(Build.SourcesDirectory)/tools/ci_build/github/js/react_native_e2e_full_ios_framework_build_settings.json
-      cd $(Build.BinariesDirectory)/ios_framework/framework_out
-      zip -r onnxruntime-c.zip .
+      $(Build.SourcesDirectory)/tools/ci_build/github/linux/docker/inference/x64/python/cpu/scripts/install_protobuf.sh -d $(Build.SourcesDirectory)/cmake/deps.txt -p $(Build.BinariesDirectory)/protobuf_install
+      python $(Build.SourcesDirectory)/tools/ci_build/github/apple/build_and_assemble_ios_pods.py \
+        --build-dir $(Build.BinariesDirectory)/ios_framework_full \
+        --staging-dir "$(Build.BinariesDirectory)/staging" \
+        --pod-version "1.15.0" \
```

Comment:
temporarily hard-coded pod-version here, is there a place we store pipeline version variables?